cargo-cyclonedx/src/cli.rs

```diff
@@ -64,6 +64,10 @@ pub struct Args {
         value_name = "FILENAME_PREFIX"
     )]
     pub output_prefix: Option<String>,
+
+    /// Use strict or lax mode
+    #[clap(long = "license-strict")]
+    pub license_strict: bool,
```

Comment:
I would try to keep the argument. Switching to strict by default, I would suggest `--license-lax`. Because, this also doesn't allow any invalid licenses. Using an invalid license requires the use of the `accept_named` option.